src/main/java/service/CsvReaderService.java

```diff
@@ -0,0 +1,29 @@
+package service;
+
+import core.basesyntax.FruitTransaction;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvReaderService {
+    public List<FruitTransaction> readTransactions(String filePath) throws IOException {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new FileReader(filePath))) {
+            String line;
+            br.readLine();
+            while ((line = br.readLine()) != null) {
+                String[] parts = line.split(",");
+                String type = parts[0].trim();
+                String fruit = parts[1].trim();
+                int quantity = Integer.parseInt(parts[2].trim());
```

Comment:
avoid using magic numbers. 

indexes should always be constant with proper name.